components/purchase_modal/purchase_modal.tsx

```diff
@@ -284,18 +285,54 @@ function DelinquencyCard(props: DelinquencyCardProps) {
                     </button>
                 </div>
                 <div className='plan_billing_cycle delinquency'>
-                    <FormattedMessage
-                        defaultMessage={
-                            'Upon reactivation you will be charged the total owed.  Your bill is calculated at the end of the billing cycle based on the number of enabled users.'
-                        }
-                        id={'cloud_delinquency.cc_modal.disclaimer'}
-                    />
-                    <a onClick={seeHowBillingWorks}>
+                    {Boolean(!props.isCloudDelinquencyGreaterThan90Days) && (
+                        <FormattedMessage
+                            defaultMessage={
+                                'Upon reactivation you will be charged the total owed.  Your bill is calculated at the end of the billing cycle based on the number of enabled users. {seeHowBillingWorks}'
```

Comment:
```suggestion
                                'When you reactivate your subscription, you'll be billed the total outstanding amount. Your bill is calculated at the end of the billing cycle based on the number of enabled users. {seeHowBillingWorks}'
```

Reply:
@nickmisasi, is the total amount billed immediately? Or is it billed at the end of the cycle?